index.html

```diff
@@ -825,6 +825,28 @@ <h2>
           for a web page using CSS.
         </p>
       </section>
+      <section class='informative'>
+        <h2>
+          Interaction with Web Content Accessibility Guidelines
+        </h2>
+        <p>
+            The Web Content Accessibility Guidelines 2.1 specification [[WCAG21]] includes a Success Criterion (<a href="https://www.w3.org/TR/WCAG21/#orientation">SC 1.3.4</a>) related to screen orientation.
+        </p>
+
+        <p>
+            ​The intent of this Success Criterion is to ensure that all content and functionality is available regardless of the display
+            orientation (portrait or landscape). Some websites and applications automatically set the screen to a particular display
+            orientation and expect that users will respond by rotating their device to match.
+        </p>
+
+        <p>
+            However, some users may have their devices mounted in a fixed orientation (e.g. on the arm of a power wheelchair). Therefore,
+            websites and applications need to support both orientations by making sure content and functionality is available in
```

Comment:
Also add "essential" into this sentence: "...Add line comment
+            However, some users may have their devices mounted in a fixed orientation (e.g. on the arm of a power wheelchair). Therefore,  
844
Add line comment
+            websites and applications need to support both orientations by making sure essential content and functionality is..."